src/main.py

```diff
@@ -129,13 +118,13 @@ async def handle_event(event: Dict, event_context, projects_ids: Optional[List[s
             scheduled_execution_id=context.scheduled_execution_id
         )
 
-        if not projects_ids:
-            projects_ids = await get_all_accessible_projects(context, gcp_session, token)
+        projects_ids = await get_all_accessible_projects(context, gcp_session, token)
 
         disabled_projects, disabled_apis_by_project_id = await get_disabled_projects_and_disabled_apis_by_project_id(context, projects_ids)
 
         if disabled_projects:
-            context.log(f"monitoring.googleapis.com API disabled in the projects: " + ", ".join(disabled_projects) + ", that projects will not be monitored")
+            context.log(f"Cannot monitor projects: {disabled_projects}. "
+                        f"Enable required services to do so: {REQUIRED_SERVICES}")
```

Comment:
Is this the only reason (not having required services) for projects being disabled?

Reply:
Even if it is, I would log it somewhere near that logic. Because if this logic changes, I dont think anyone would remember about this log line here that gives reason

Reply:
Now at least, yes